.github/workflows/CI.yml

```diff
@@ -19,13 +19,20 @@ jobs:
           - 10
           - 12
           - 14
+        node_arch:
+          - x64
+        include:
+          - node: 12
```

Comment:
Typo? This is `node` whereas above it is `node_version`?

Reply:
This typo could cause the task to not actually install a new copy of Node, if I understand correctly.

 From: https://github.com/actions/setup-node#v2-beta

> The `node-version` input is optional.  If not supplied, node which is in your PATH will be used.

Reply:
```suggestion
          - node_version: 12
```

Reply:
@aminya this PR works with the small typo fix I posted above.

https://github.com/DeeDeeG/apm/runs/1077130953#step:5:308

Reply:
I added a warning in https://github.com/actions/setup-node/pull/194/commits/2a5c060ce28fea5df265375fde1fa37b767a310b